aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,84 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+   - Provided guidance and examples on Sidebar for Side-to-Side Navigation. [Figma](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9562) [Design System site](https://design-system.hpe.design/templates/side-to-side-navigation?q=side)
+   - Finalized ideating and documentation for Matrix Navigation. [Figma](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9780) and published on [Design System site](https://design-system.hpe.design/templates/matrix-navigation)
+   - Finalized ideation for Global Header within [Side-to-Side Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9562)
+- **Page Layouts** 
+   - Commenced an overview of [Page Layouts that provides definitions and an anatomy.](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=1876%3A16251)
+   - Commenced ideation and documentation such as [responsive design, breakpoints, and padding on containers within Page Layouts.](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=1876%3A16251)
+
+## November 22 - December 3, 2021
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **Button** 
+   - Added guidance on the ordering of buttons on a form, layer.[Figma](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=667%3A385) published on [Design System site](https://design-system.hpe.design/components/button?q=button#single-buttons)
+   - Commenced Button Guidance on Cancel Button being primary or secondary 
+   - Completed guidance for [button positioning on wizards and modals.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4685%3A9517)
+   - Guidance on [left vs right aligned buttons.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4685%3A9536)
+   - Re-visited [hover state for primary button and updated color.](https://github.com/grommet/hpe-design-system/issues/1271)
+- **Navigation** 
+   - Commenced [explorations for Reverse Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18)
+   - Commenced [exploration of Matrix Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A18)
+### Stay current with the latest changes
+- Added [trash icon to FileInput validation example](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=1164%3A43348)
+- Changed the [cog icon on ordering of items to column icon based on a user study conducted.](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items?node-id=33%3A12)
+- Added resize vertical to textarea, this allows users to resize the [TextArea](https://design-system.hpe.design/components/textarea?q=text)
```

Comment:
We can likely omit this -- this was just fixing an example but not anything related to design system guidance

Reply:
done